platforms/core-runtime/java-language-extensions/src/main/java/org/gradle/internal/exceptions/DefaultMultiCauseException.java

```diff
@@ -102,7 +107,7 @@ public List<? extends Throwable> getCauses() {
     }
 
     @Override
-    public Throwable initCause(Throwable throwable) {
+    public synchronized Throwable initCause(Throwable throwable) {
```

Comment:
💭 According to ErrorProne this should be `synchronized` since the overridden method is also `synchronized`.